pyproject.toml

```diff
@@ -15,8 +15,9 @@ skip = ["cloudinit/cmd/main.py", ".tox", "packages", "tools"]
 
 [tool.mypy]
 follow_imports = "silent"
-warn_unused_ignores = "true"
+check_untyped_defs = true
```

Comment:
Nit: Can we make types of the booleans consistent? `true` on this line but `"true"` on the following lines is a little weird.

Reply:
Done.